docs/src/featureToggle.js

```diff
@@ -2,5 +2,5 @@
 module.exports = {
   enable_website_banner: false,
   enable_toc_banner: true,
```

Comment:
Maybe we could hide the job ad. I don't recall we saw much relevant applications, it feels a bit spammy.

```suggestion
  enable_toc_banner: false,
```

Reply:
I tried this change, but this switch controls only the Ukrania support banner. There's no Feature toggle for the Job Ad.
It may make sense to do a PR exclusive for this.

Reply:
https://github.com/mui/material-ui/pull/36080